hail/python/hail/docs/datasets/schemas/gnomad_genome_sites.rst

```diff
@@ -3,7 +3,7 @@
 gnomad_genome_sites
 ===================
 
-*  **Versions:** 2.1.1, 3.1
+*  **Versions:** 2.1.1, 3.1, 3.1.1, 3.1.2
```

Comment:
Can you paste in the schema for, say, 3.1.2. We haven't been consistent about pasting every schema in. I'm not sure that's a sustainable strategy long term anyway, but including the most recent version seems like a reasonable choice.